src/pybind/mgr/snap_schedule/fs/schedule.py

```diff
@@ -296,7 +296,7 @@ def rm_schedule(cls,
                              (path,))
             row = cur.fetchone()
 
-            if len(row) == 0:
+            if not row:
```

Comment:
```suggestion
            if row is None:
```

better off being more explicit.

Reply:
> better off being more explicit.

the same style in add_retention/rm_retention, do we need to change them all?

https://github.com/ceph/ceph/blob/4235e99e593f41c3a2fddb5b2040a23e0d5974ec/src/pybind/mgr/snap_schedule/fs/schedule.py#L348

https://github.com/ceph/ceph/blob/4235e99e593f41c3a2fddb5b2040a23e0d5974ec/src/pybind/mgr/snap_schedule/fs/schedule.py#L372

Reply:
one thing at a time. you could have a dedicated cleanup commit in this PR.

Reply:
> one thing at a time. you could have a dedicated cleanup commit in this PR.

changed